part7/anecdotes/src/components/CreateNew.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+
+const CreateNew = ({ addNew, history }) => {
+    const [content, setContent] = useState('')
+    const [author, setAuthor] = useState('')
+    const [info, setInfo] = useState('')
+
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        addNew({
+            content,
+            author,
+            info,
+            votes: 0
+        })
+    
+        // I DONT GET IT
+        history.push('/')
```

Comment:
ReactRouter `history` pakārtojās šim: https://developer.mozilla.org/en-US/docs/Web/API/History_API

- via: https://github.com/ReactTraining/history/blob/master/docs/getting-started.md#user-content-intro:~:text=A%20%22browser%20history%22%20is%20for%20use,HTML5%20history%20API%20(see%20cross%2Dbrowser%20compatibility)
- via: https://reactrouter.com/web/api/history

History fundamentāli ir Stack, jeb LIFO (Last In, First Out) aka its inverse FILO (First In, Last Out) datu kolekcija:

- https://en.wikipedia.org/wiki/Stack_(abstract_data_type)
- https://everythingcomputerscience.com/discrete_mathematics/Stacks_and_Queues.html

Attiecīgi, ja tu Chrome uzspiedīsi ilgāk uz back pogas, tevi prezentēs ar visiem history ierakstiem kuru konkrētā sesija/tabs saprot:
![image](https://user-images.githubusercontent.com/1030080/100221383-49972300-2f21-11eb-814e-edd3a9a76473.png)
Tādā veidā revealojot History as a Stack nature.

Līdz ar to, `push` ieliek History Stack'ā jaunu ierakstu, vienlaicīgi arī pārslēdzoties uz to. Un `/` ir ceļš tavā browser navigation bārā, kur '/' vienmēr atsaucās uz root, jeb ja atradies `http://localhost/anecdotes/1`, tad '/' ir ekvivalents `http://localhost/`.